src/main/java/com/ericsson/ei/rules/RulesHandler.java

```diff
@@ -39,57 +45,105 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 @Component
-@Scope(value="thread", proxyMode = ScopedProxyMode.TARGET_CLASS)
+@Scope(value = "thread", proxyMode = ScopedProxyMode.TARGET_CLASS)
 public class RulesHandler {
     private static final Logger LOGGER = LoggerFactory.getLogger(RulesHandler.class);
 
-    @Value("${rules.path}") private String jsonFilePath;
+    private static final String EI_HOME_DEFAULT_NAME = ".eiffel-intelligence";
+
+    @Value("${rules.path}")
```

Comment:
I'd like a more clear comment in the application.properties file about this flag, because now it only explains the property in itself, not if it's a relative or full path. And if we add URL support then this should be updated. If string is prefixed with http:// or file:// it is clear, but if no prefix is used, then it should be clear we are assuming relative file path.

If it's too much text to put in the comments in application.properties, then we should link to GitHub documentation about configuration of EI.

Reply:
I added a small comment in the properties file. I haven't yet started looking into the whole file:// thing though.

Reply:
You can update documentation regarding rules.path property in markdown documentations: https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/configuration.md

Reply:
I'll update the doc when the PR is final =)

Reply:
Done.